src/styles/team.scss

```diff
@@ -182,29 +186,34 @@ $secondColor: white;
     }
 }
 .ballPict{
-    position: absolute;top: 0; left: 40px;
+    position: absolute;
+    top: 0; 
+    left: 40px;
 }
 .ballArrowPict{
-    position: absolute; bottom: -20px; right: -20px;
+    position: absolute; 
+    bottom: -20px; 
+    right: -20px;
 }
 .hooks{
     color: #f5f5f5 !important;
```

Comment:
It's a bad practice to use !important flag. It's better to avoid it by nesting and adding classes. 

And we have the variable for white color.